data/input/oneshot_67p_opengl.json

```diff
@@ -76,13 +63,30 @@
                 "file": "data/models/67P_C-G_shape_model_MALMER_2015_11_20-in-km.obj",
                 "name": "67P"
             },
-            "albedo": 0.15,
```

Comment:
Is it intended that the albedo value is deleted?

Reply:
Referring to comment reply at `oneshot_itokawa_opengl.json`